src/Main.java

```diff
@@ -43,9 +54,14 @@ public static void main(String[] args) {
         System.out.println("Перевес на " + (boxer2Weight % boxer1Weight) + " кг");
 
         // задание 8
-        System.out.println("Количество часов = " + 640 + " часов");
-        System.out.println("Человек работает = " + 8 + " часов");
-        System.out.println("Всего работников в компании — " + (640 / 8) + " человек");
-        System.out.println("Если в компании работает " + (640 / 8 + 94) + " человек, то всего " + ((640 / 8 + 94) * 8) + " часов работы может быть поделено между сотрудниками");
+        var workHours = 640;
+        System.out.println("Количество часов = " + workHours + " часов");
+        var humanWork = 8;
+        System.out.println("Человек работает = " + humanWork + " часов");
+        var people = workHours / humanWork;
+        System.out.println("Всего работников в компании — " + people + " человек");
+        people = people + 94;
+        workHours = people * 8;
```

Comment:
почему 8-то? надо использовать переменные, а не числа
```suggestion
        workHours = people * humanWork;
```